pinot-broker/src/main/java/org/apache/pinot/broker/broker/BrokerAdminApiApplication.java

```diff
@@ -109,6 +112,8 @@ protected void configure() {
         bind(brokerConf.getProperty(CommonConstants.Broker.CONFIG_OF_BROKER_ID)).named(BROKER_INSTANCE_ID);
         bind(serverRoutingStatsManager).to(ServerRoutingStatsManager.class);
         bind(accessFactory).to(AccessControlFactory.class);
+        bind(instanceId).named(BrokerAdminApiApplication.BROKER_INSTANCE_ID);
+        bind(pinotBrokerHealthCheck).to(PinotBrokerHealthCheck.class);
```

Comment:
This is to register objects with the DI framework because these were instantiated manually. As mentioned in the description in order to preserve the start time, it is essential to create an object at start time and ensure that it exists through out the lifecycle of the application